docs/theme/base.html

```
@@ -0,0 +1,118 @@
+        <a href="{{ 'community/contributing/' | url }}">Contributing</a>
+      </div>
+    </div>
+  </footer>
+</div>
+
+<!-- Search overlay -->
+<div class="ld-search" id="search-overlay" hidden>
+  <div class="ld-search__scrim" data-search-close></div>
+  <div class="ld-search__panel" role="dialog" aria-label="Search documentation">
+    <div class="ld-search__bar">
+      <input class="ld-search__input" id="search-input" type="search" placeholder="Search documentation…" autocomplete="off" spellcheck="false">
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)